core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -201,7 +202,7 @@ public final void doBuild(StaplerRequest2 req, StaplerResponse2 rsp, @QueryParam
         }
 
         if (!asJob().isBuildable()) {
-            throw HttpResponses.error(SC_CONFLICT, new IOException(asJob().getFullName() + " is not buildable"));
+            throw HttpResponses.errorWithoutStack(SC_CONFLICT, asJob().getFullName() + " is not buildable");
```

Comment:
Also, these 409 responses were including a stack trace, which seemed inappropriate and unnecessary.